content/blog/trungnt2910/2024-08-18_[gsoc_2024]_debugging_final_report.md

```diff
@@ -0,0 +1,138 @@
++++
+type = "blog"
+title = "[GSoC 2024] Improving the Userland Debugging Experience - Final Report"
+author = "trungnt2910"
+date = "2024-08-18"
+tags = ["haiku", "software", "gsoc", "gsoc2024", "gdb", "gdbserver"]
++++
+
+# Project overview
+
+A part of Google Summer of Code 2024, this project aims to improve the userland debugging experience
```

Comment:
Shouldn't this be in past tense?

Reply:
Whoops. Fixed that and a few following parts as well.